go

```diff
@@ -85,6 +89,15 @@ def ci_build
   puts 'Done!'
 end
 
+def server_build
```

Comment:
What's the use case for `server_build`? Should it be used in `fabfile.py`? Also, both `server_build` and `update_submods` should have entries in the `COMMANDS` array. (You can steal the `CommandGroup` class from the Hub's `go` if you want. Or maybe it's time I wrote a `go` gem. :-P)